pkg/plugins/runtime/k8s/controllers/configmap_controller.go

```diff
@@ -42,6 +43,10 @@ func (r *ConfigMapReconciler) Reconcile(req kube_ctrl.Request) (kube_ctrl.Result
 	}
 
 	if err := r.VIPsAllocator.CreateOrUpdateVIPConfig(mesh); err != nil {
+		if kube_apierrs.IsConflict(err) {
+			r.Log.V(1).Info("resource conflict", "err", err)
```

Comment:
sounds scary. This should be more descriptive like
```
r.Log.V(1).Info("VIPs were updated in the other place. Retrying")
```

Reply:
also on V1 "updating VIPs for mesh" and after the operation "VIPs for mesh udpated

Reply:
Agree, fixed these